org.opent2t.sample.binaryswitch.superpopular/com.wink.binaryswitch/js/thingTranslator.js

```diff
@@ -88,7 +88,7 @@ class Translator {
 
         // Initialize Wink Helper
         winkHelper = new WinkHelper(device.props.access_token);
-        console.log('Javascript and Wink Helper initialized : ');

```

Comment:
>console [](start = 8, length = 7)

At some point do we intend to have a logging library at all so we can say log to file, gather telemetry etc or does that stuff not apply in this space?